internal/impl/devbox.go

```diff
@@ -455,6 +455,26 @@ func (d *Devbox) GenerateDockerfile(force bool) error {
 	return nil
 }
 
+// generates a .envrc file that makes direnv integration convenient
+func (d *Devbox) GenerateEnvrc(force bool) error {
+	envrcfilePath := filepath.Join(d.configDir, ".envrc")
+	filesExist := plansdk.FileExists(envrcfilePath)
```

Comment:
`fileutil.Exists`?

Reply:
I didn't know about `fileutil.Exists` we use plansdk.FileExists in so many different places that I had to keep it in planner sdk code when refactoring. 
Will have to make a refactor on all instances later on